scripts/schedule_maintenance.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+#run with "source schedule_maintenance.sh [arg1] [arg2] [arg3]"
+#pass dates as "m/d/y@hour:min"
+if (( $# > 0 ))
+then
+  if (($[$1 == true] || $[$1 == false]))
+  then
+    export IS_MAINTENANCE_SCHEDULED=$1
+    echo $IS_MAINTENANCE_SCHEDULED
```

Comment:
Maybe change this to a proper log message like `Scheduled Maintenance Message has been enabled/disabled`?

Reply:
okay. I'll change the script responses.

Reply:
and yes the warning displays on home page, about us page, and graph page.